README.md

```diff
@@ -9,51 +9,37 @@ In order to deploy the Marketplace Operator, you must:
+- `configMap` is used in `configmap` type `CatalogSources` to refer to the `ConfigMap` that backs the registry.
 
-The creation of an `OperatorSource` results in the creation of an OLM `CatalogSource` in the same namespace the marketplace operator is running in. This `CatalogSource` will be populated with operators from the `OperatorSource` ready to be managed by OLM.
+- `displayName`, `description`, `icon` and `publisher` are optional and only needed for UI purposes.
 
-#### CatalogSourceConfig
+Please see [here][community-operators] for an example `CatalogSource`.
 
-`CatalogSourceConfig` is used to create OLM `CatalogSources` consisting of operators from one `OperatorSource` so that these operators can then be managed by OLM.
+If you want an `CatalogSource` to work with private registry repositories, please take a look at the [Private Repo Authentication](docs/how-to-authenticate-private-repositories.md) documentation.
 
-Here is a description of the spec fields:
-
-- `targetNamespace` is the namespace that OLM is watching. This is where the resulting `CatalogSource`, which will have the same name as the `CatalogSourceConfig`, is created or updated.
-
-- `source` is the name of the `OperatorSource` that the packages originate from.
-
-- `packages` is a comma separated list of operators.
-
-- `csDisplayName` and `csPublisher` are optional but will result in the `CatalogSource` having proper UI displays.
+On adding a `CatalogSource` to an OKD cluster, operators will be visible in the [OperatorHub UI](https://github.com/openshift/console/tree/master/frontend/public/components/operator-hub) in the OKD console. There is no equivalent UI in the Kubernetes console.
 
-To learn more about how the Marketplace Operator resolves source and package combinations, please review [this doc](docs/csc-source-resolution.md).
-
-Please see [here](deploy/examples/catalogsourceconfig.cr.yaml) for an example `CatalogSourceConfig`.
-
-Once a `CatalogSourceConfig` is created successfully you can create a [`Subscription`](https://github.com/operator-framework/operator-lifecycle-manager#discovery-catalogs-and-automated-upgrades) for your operator referencing the newly created or updated `CatalogSource`.
-
-Please note that the Marketplace operator uses `CatalogSourceConfigs` and `CatalogSources` internally and you will find them present in the namespace where the Marketplace operator is running. These resources can be ignored and should not be modified or used.
+Once a `CatalogSource` is created successfully you can create a [`Subscription`](https://github.com/operator-framework/operator-lifecycle-manager#discovery-catalogs-and-automated-upgrades) for your operator referencing the newly created or updated `CatalogSource`.
```

Comment:
This should link to https://olm.operatorframework.io/docs/tasks/install-operator-with-olm/#install-your-operator